scripts/install-chef-suse.sh

```diff
@@ -252,13 +254,20 @@ You should also check the Installation Manual, in
 particular the Troubleshooting section.  Note that this script can
 safely be re-run multiple times if required.
 EOF
+        touch /opt/dell/crowbar_framework/.crowbar-install-failed
     else
         kill_spinner
     fi
 }
 
 trap exit_handler EXIT
+trap post_fail_handler INT
 
+post_fail_handler ()
+{
+    touch /opt/dell/crowbar_framework/.crowbar-install-failed
```

Comment:
then we should move `.crowbar-installed-ok` too, that's why i kept it there. you agree with that?